docs/configuration/sample-config.md

```diff
@@ -2,39 +2,100 @@
 id: sample-config
 title: Server Config File
 ---
+# Server configuration file
+Starting the server with various configuration options can be done using a configuration file instead of using just flags.  
+The command used to start the server with config file `polygon-edge server --config <config_file_name>`
 
-Following is the sample format for configuration file: 
+## Export config file with default configuration
+Config file with default settings for the Polygon Edge server can be exported into a config file in either `yaml` or `json` file format.
```

Comment:
I believe you meant to say `The configuration with default settings...`

Reply:
Fixed 1713f6a24cb0da0f003c85013685ea41d8b92152